core/src/main/java/com/sk89q/worldedit/command/UtilityCommands.java

```diff
@@ -524,6 +514,17 @@ public void remove(Actor actor, CommandContext args) throws WorldEditException,
     @CommandPermissions("worldedit.calc")
     public void calc(final Actor actor, @Text String input) throws CommandException {
         try {
+            FaweLimit limit;
+            FawePlayer player = null;
+            if (actor != null) {
+                player = FawePlayer.wrap(actor);
+            }
+            if (player == null) {
+                limit = FaweLimit.MAX;
+            } else {
+                limit = player.getLimit();
+            }
+
```

Comment:
You don't need to check if it's null, there will always be a Actor, and FawePlayer.wrap(...) will always wrap the player object (event console)